cmd/minikube/cmd/start.go

```diff
@@ -123,6 +124,7 @@ func init() {
 	startCmd.Flags().String(serviceCIDR, pkgutil.DefaultServiceCIDR, "The CIDR to be used for service cluster IPs.")
 	startCmd.Flags().StringSliceVar(&insecureRegistry, "insecure-registry", nil, "Insecure Docker registries to pass to the Docker daemon.  The default service CIDR range will automatically be added.")
 	startCmd.Flags().StringSliceVar(&registryMirror, "registry-mirror", nil, "Registry mirrors to pass to the Docker daemon")
+	startCmd.Flags().String(imageRepository, "", "Image repository to pull down docker images.")
```

Comment:
The help text isn't fully helpful here. How about: 

"Alternative image repository to pull docker images from."

Reply:
If there are any publicly available examples of mirrors that work -- especially within China, could you add them as an example?

Reply:
Sure. I'll change the help text with mirror repository example for China users.